README.md

```diff
@@ -366,6 +366,8 @@ The `templates/user.yaml` file has been configured to grant `data_upload` privil
 > DATA_UPLOAD_BUCKET: 'bucket1'
 ```
 
+> 🟢 Note: Any upload buckets including local machines are currently not supported unless they are S3 compliant.
```

Comment:
Is it better to say that for `local machines` uploads aren't supported at all? Not that it's _impossible_, but it's also not something that works out of the box

Reply:
there are probably solutions for s3-compliant local storage but i never looked into it
but yeah maybe we can say "not supported out of the box" instead of "not supported" @xritter1?

Reply:
sounds good! Review please again. Merci